src/licensedcode/plugin_license.py

```diff
@@ -167,7 +177,8 @@ def setup(self, **kwargs):
         loaded index.
         """
         from licensedcode.cache import populate_cache
-        populate_cache()
+        additional_directories = kwargs.get('external_license_directory')
+        populate_cache(additional_directories=additional_directories)
```

Comment:
IMHO additional_directories should be used only when reindexing and not in general.